tests/test_node_util.py

```diff
@@ -142,10 +141,6 @@ def test_json_error(complex_parameter_node):
     parameter._json_attrs = replace(parameter._json_attrs, value="abc")
     with pytest.raises(CRIPTNodeSchemaError):
         parameter.validate()
-    # Let's break it completely
-    parameter._json_attrs = None
```

Comment:
I removed this test.
It was pretty artificial and didn't work with the new way anymore.

Reply:
just be sure that the code you are putting in will not break anything and it is HEAVILY tested if possible